app/src/main/java/com/gnag/example/MainActivity.java

```
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2016 Bryan Kelly
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.gnag.example;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        String test = "";
+        String testCompare = "Fail";
+
+        if (test == testCompare) {
+            Log.e("TEST", "Total Fail");
+        }
+
+        try {
+            Log.e("TEST", "Empty");
+        } catch (Exception e) {
+
+        } finally {
```

<b>Reporter: </b>PMD<br /><b>Violation: </b><a target="_blank" href="https://pmd.github.io/pmd-5.4.1/pmd-java/rules/java/empty.html#EmptyCatchBlock">EmptyCatchBlock</a><br /><b>Notes: </b>Avoid empty catch blocks